docs/jackson/examples.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 4
+---
+
+# Examples & Resources
+
+## Examples
+
+- [Jackson with Supertokens](https://github.com/devkiran/jackson-supertokens)
+- [Jackson with Next.js](https://github.com/boxyhq/jackson-demo)
+- [Jackson with Express](https://github.com/devkiran/express-jackson-demo)
```

Comment:
Can you move this to boxyhq org and link to that please

Reply:
Done.